Mastermind/Mastermind/Views/GameboardView.swift

```diff
@@ -9,10 +9,13 @@ import SwiftUI
 
 struct GameboardView: View {
     @ObservedObject var viewModel: MastermindViewModel //um die Spiellogik auszuführen
+
+    @State private var isRowLocked = false //wird true, wenn der Submit-Button angeklickt wird
```

Comment:
Following the guidelines in Cariad, after the //, we should add a space. it helps for readability. 
```suggestion
    @State private var isRowLocked = false // wird true, wenn der Submit-Button angeklickt wird
```

You can apply this rule for every comments.